src/lib/censorRecords.js

```diff
@@ -69,8 +69,8 @@ const censorDeath = (death, roles) => {
       marginalNote: censorField(false, death.status.marginalNote),
       onAuthorityOfRegistrarGeneral: censorField(true, death.status.onAuthorityOfRegistrarGeneral)
     },
-    previousRegistration: death.previousRegistration && censorRecords(death.previousRegistration, roles),
-    nextRegistration: death.nextRegistration && censorRecords(death.nextRegistration, roles),
+    previousRegistration: death.previousRegistration && censorDeath(death.previousRegistration, roles),
+    nextRegistration: death.nextRegistration && censorDeath(death.nextRegistration, roles),
```

Comment:
Was this a bug that we missed?

Reply:
Not sure I get what you're asking but it was originally removed due to a build failure, refactored but then it turned out to be a different problem and was subsequently added back.

Reply:
Well there is no `censorRecords` function is there? So the old code would crash in the presence of a chain and this is a bug-fix that we should publish ASAP.